.github/CODEOWNERS

```diff
@@ -61,6 +61,7 @@
 /core/storage/opportunity/ @DubeySandeep @sagangwee
 /core/templates/domain/opportunity @DubeySandeep @sagangwee
 /core/templates/pages/community-dashboard-page/ @DubeySandeep @sagangwee
+/core/templates/services/image-local-storage*.ts @DubeySandeep
```

Comment:
@DubeySandeep Is this fine, or should I be codeowner?

Reply:
I'm fine with owning this, but can you move it to the RTE project section? [As it's used for RTE]

Reply:
Can you please move this to the correct place?

Reply:
Done